yarn/pom.xml

```diff
@@ -148,20 +148,25 @@
       <scope>test</scope>
      </dependency>
      <dependency>
-       <groupId>com.sun.jersey</groupId>
-       <artifactId>jersey-core</artifactId>
+       <groupId>org.glassfish.jersey.core</groupId>
```

Comment:
Actually this doesn't work because the Mini YARN cluster needs Jersey 1 to start up. Since this is only in test scope we should be fine to keep the Jersey 1 dependency here - I'll add a comment to clarify that.